src/commands/runScripts/start-dev.js

```diff
@@ -19,9 +19,26 @@ const ready = {
 };
 let appServerProcess;
 
-const log = message => console.log(chalk.yellow('>>'), message);
+const log = (...msgs) => console.log(chalk.yellow('>>'), ...msgs);
+
+// regex to find all instances of the current working directory as a string
```

Comment:
took me some time to parse this comment. each line starts with `process.cwd` and we wish to strip it, correct?

imo, this code is very clear by itself, but what its doing is harder to understand.